tests/integration/api_packages_pub_test.go

```diff
@@ -119,8 +119,6 @@ description: ` + packageDescription
 		pb, err := packages.GetBlobByID(db.DefaultContext, pfs[0].BlobID)
 		assert.NoError(t, err)
 		assert.Equal(t, int64(len(content)), pb.Size)
-
-		resp = uploadFile(t, result.URL, content, http.StatusBadRequest)
```

Comment:
This seems a single test, and it should be kept. `_ = uploadFile(t, result.URL, content, http.StatusBadRequest)`

Reply:
done